packages/web-client/app/components/card-pay/balance-chooser-dropdown/index.css

```diff
@@ -28,11 +29,13 @@
   overflow: hidden;
 }
 
-.balance-chooser-dropdown .ember-basic-dropdown-content.ember-basic-dropdown--transitioning-in {
+.balance-chooser-dropdown
+  .ember-basic-dropdown-content.ember-basic-dropdown--transitioning-in {
   animation: drop-fade-below var(--boxel-transition);
 }
 
-.balance-chooser-dropdown .ember-basic-dropdown-content.ember-basic-dropdown--transitioning-out {
+.balance-chooser-dropdown
+  .ember-basic-dropdown-content.ember-basic-dropdown--transitioning-out {
```

Comment:
The underlying ember-basic-dropdown is leaking through our abstraction here. Maybe we should consider introducing CSS variables for this?

Reply:
Will do on next PR, I noticed we have this animation spread across the components